engine/src/button.cpp

```diff
@@ -1437,7 +1437,7 @@ Boolean MCButton::mup(uint2 which, bool p_release)
 	}
 	if (state & CS_GRAB)
 	{
-		if (flags && F_AUTO_HILITE)
+		if (flags & F_AUTO_HILITE)
```

Comment:
Yes, I'm planning on reviewing which errors I can turn on at the end and whether they can be turned on for all builds or just Mac.